components/MainNavigator.tsx

```diff
@@ -3,7 +3,12 @@ import { Nav, Navbar } from 'react-bootstrap';
 
 export const MainNavigator: FC = () => {
   return (
-    <Navbar expand="lg" bg="success" data-bs-theme="dark" className="shadow-sm">
+    <Navbar
+      expand="lg"
+      bg="success"
+      data-bs-theme="dark"
+      className="shadow-sm sticky-top"
```

Comment:
```suggestion
    <Navbar
      className="shadow-sm"
      bg="success"
      data-bs-theme="dark"
      sticky="top"
      expand="lg"
      collapseOnSelect
```
1. https://react-bootstrap.github.io/docs/components/navbar#sticky-top
2. https://react-bootstrap.github.io/docs/components/navbar#responsive-behaviors